mkdocs.yml

```diff
@@ -116,7 +201,8 @@ nav:
         - 'Resizing a VM': 'technical-documentation/resizing_vm.md'
       - 'Checking Coding Standards': 'technical-documentation/checking-coding-standards.md'
       - 'Contributing Workflow': 'contributing/contributing-workflow.md'
-      - 'Editing Documentation': 'contributing/editing-docs.md'
```

Comment:
This is @manez 's new tutorial for people who haven't made a PR before, can we keep it in?

Reply:
Good eye, I must have lost that one with all the reshuffling. I'll push an update in a bit